### ISSUE TITLE
Support Scikit Learn Pipeline

### ISSUE THREAD

**Issue by danuker** [danuker/symreg#10]
Since SymReg works best with normalized data, it would be ideal to allow it to be used in a sklearn pipeline, with a [`StandardScaler(with_mean=False)` preprocessor](https://scikit-learn.org/stable/modules/preprocessing.html#preprocessing-scaler).

Make sure this is possible, and add an example to the README.

----

Alternatively (if Pipeline support needs sklearn dependency): have an argument to just normalize the data inside. Divide by the median of the data for the regression, then multiply back when returning the resulting functions.
